ui/search.js

```diff
@@ -53,17 +58,48 @@ function removeCloseButton() {
     delete searchResults._closeButton;
 }
 
-function setSearchResultsXAlign(align) {
+function addMaxWidthBoxInSearch() {
     const { viewSelector } = Main.overview;
-    viewSelector._searchResults.x_align = align;
+
+    viewSelector._searchPage.remove_child(viewSelector._searchResults);
```

Comment:
Can this object be removed when removed from parent? Should we keep a ref to it before removing?

Reply:
No need to manual ref management on GJS.

Reply:
Ok, I thought the object could be deleted if the ref count would drop to 0.